server/testutils/factories/withdrawablesFactory.ts

```diff
@@ -0,0 +1,10 @@
+import { Factory } from 'fishery'
+import { Withdrawables } from '../../@types/shared'
+
+export default Factory.define<Withdrawables>(() => ({
+  notes: [
+    '1 or more placements cannot be withdrawn as they have an arrival',
+    '1 or more placements cannot be withdrawn as they have an arrival recorded in Delius',
+  ],
+  withdrawables: [],
```

Comment:
We should add some withdrawables here rather than having an empty array then we don't have to separately create the withdrawables every time we use this factory.